src/main/resources/proto/ga4gh/metadata.proto

```diff
@@ -0,0 +1,108 @@
+syntax = "proto3";
+
+package ga4gh;
+
+import "proto/google/protobuf/struct.proto";
```

Comment:
If it works I don't have any real objection to either. IIRC, we put proto there to avoid conflicts with the package "google" which had already been imported into the reference server.

I don't know the history around main/resources. Removal of those intermediate folders seems reasonable at a glance, though I don't know why they exist in this form currently.